Documentation/contributing/development/images.rst

```diff
@@ -198,85 +198,49 @@ Image dependency:
+   to approve the build that was created by GitHub Actions `here <https://github.com/cilium/cilium/actions?query=workflow:%22Base+Image+Release+Build%22>`_.
 
-.. image:: ../../images/cilium-quayio-tag-2.png
-    :align: center
+4. Wait for the build to complete. Only after the image is available run:
 
-4. After enabling the trigger, click again on the wheel.
-5. And click on "Run Trigger Now".
+    $ make -C images/ update-runtime-image update-builder-image
 
-.. image:: ../../images/cilium-quayio-tag-3.png
-    :align: center
+5. Commit your changes and re-push to the PR in cilium/cilium.
 
-6. A new pop-up will appear to select your desired branch.
-7. If you're interested in simply bumping the image to have the latest
-   packages, then select the release branch (i.e. v1.7, v1.8). If you already
-   have a branch that contains changes, select the branch that contains the new
-   changes.
+.. code:: bash
 
-.. image:: ../../images/cilium-quayio-tag-4.png
-    :align: center
+    $ git commit --amend -s images/ -m "update cilium-{runtime,builder}"
 
-8. After selecting your branch click on "Start Build".
+6. Ping one of the members of `team/build <https://github.com/orgs/cilium/teams/build/members>`_
+   to approve the build that was created by GitHub Actions `here <https://github.com/cilium/cilium/actions?query=workflow:%22Base+Image+Release+Build%22>`_.
 
-.. image:: ../../images/cilium-quayio-tag-5.png
-    :align: center
+7. Wait for the build to complete. Only after the image is available run:
 
-9. Once the build has started you can disable the Build trigger by clicking on
-   the wheel.
-10. And click on "Disable Trigger".
+    $ make -C images/ update-runtime-image update-builder-image
 
-.. image:: ../../images/cilium-quayio-tag-6.png
-    :align: center
+8. Commit your changes and re-push to the PR in cilium/cilium.
 
-11. Confirm that you want to disable the build trigger.
+.. code:: bash
 
-.. image:: ../../images/cilium-quayio-tag-7.png
-    :align: center
+    $ git commit --amend -s images/ -m "update cilium-{runtime,builder}"
 
-12. Once the build is finished click under Tags (on the left menu).
-13. Click on the wheel and;
-14. Add a new tag to the image that was built.
-
-.. image:: ../../images/cilium-quayio-tag-8.png
-    :align: center
-
-15. Write the name of the tag that you want to give for the newly built image.
-16. Confirm the name is correct and click on "Create Tag".
-
-.. image:: ../../images/cilium-quayio-tag-9.png
-    :align: center
-
-17. After the new tag was created you can delete the other tag, which is the
-    name of your branch. Select the tag name.
-18. Click in Actions.
-19. Click in "Delete Tags".
-
-.. image:: ../../images/cilium-quayio-tag-10.png
-    :align: center
-
-20. Confirm that you want to delete tag with your branch name.
-
-.. image:: ../../images/cilium-quayio-tag-11.png
-    :align: center
+9. Grep and replace all other places where the images are used. (This step will
```

Comment:
We should really be using `#.` instead of numbers directly, so that when we update the numbers like this we don't jump straight from step 9 to 21.

Reply:
They don't work because of the code-blocks:
![image](https://user-images.githubusercontent.com/5714066/108694498-6979bb00-74ff-11eb-94f8-ac4758ae6700.png)

Reply:
I discussed out-of-band with Andre, but for those following along at home: Sphinx likes 3-space indents. Apply the indentation that way and you should not see this problem.